src/apps/statements/repo/modelsRepo/getFullActivity/mongo.ts

```diff
@@ -39,6 +40,11 @@ export default (config: FacadeConfig): Signature => {
       extensions: replaceDotsInExtensions(/&46;/g, '.')(result.extensions),
       ...(result.type === undefined ? {} : { type: result.type }),
       ...(result.moreInfo === undefined ? {} : { moreInfo: result.moreInfo }),
+      ...(
```

Comment:
@PrinceWaune can we use the same line break structure defined styling for lines 41 and 42 as you've used for 43?